etna/models/tbats.py

```diff
@@ -175,9 +217,33 @@ def _decompose_forecast(self, horizon: int) -> np.ndarray:
         raw_components = np.stack(components, axis=0)
 
         if model.params.components.use_box_cox:
-            transformed_pred = np.sum(raw_components, axis=1)
-            pred = model._inv_boxcox(transformed_pred)
-            raw_components = raw_components * pred[..., np.newaxis] / transformed_pred[..., np.newaxis]
+            raw_components = self._rescale_components(raw_components)
+
+        return raw_components
+
+    def _decompose_predict(self) -> np.ndarray:
```

Comment:
In in-sample predictions we also have `residuals` component as I understand

Reply:
No, prediction formed only from components of state vector. Residuals are `y - y_hat`. ([here](https://github.com/intive-DataScience/tbats/blob/1f2e0b5e769250c8ec0604fd75ef08ebbe251d37/tbats/abstract/Model.py#L236))